File: Hayaa.BaseModel/Hayaa.Redis.Client/Config/RedisClientConfig.cs
using Hayaa.Redis.Client.Model;
using System;
using System.Collections.Generic;
using System.Text;

namespace Hayaa.Redis.Client.Config
{
    /// <summary>
    /// 服务配置类
    /// </summary>
    [Serializable]
    public class RedisClientConfig 
    {
        public List<RedisConfig> RedisConfigs { set; get;  }
      

    }
}

File: Java/hayaa.redisclient/src/main/java/hayaa/redis/client/RedisClientConfig.java
package hayaa.redis.client;
import java.util.List;

class RedisClientConfig {
    public List<RedisConfig> getRedisConfigs() {
        return RedisConfigs;
    }

    public void setRedisConfigs(List<RedisConfig> redisConfigs) {
        RedisConfigs = redisConfigs;
    }

    private List<RedisConfig> RedisConfigs;
}
